### PR TITLE
Fixed tag with Boolean/Null value trigger parse error

### PR DESCRIPTION
**Pull request by eugenegp**
'instance,bool_tag=,value_tag=value cpucount=10i,free=1i,test="string",bool=false,value=1.11':

### CODE REVIEW

File: src/InfluxDB/Point.php [hunk 1]
```diff
@@ -122,6 +122,16 @@ public function getTags()
...
     {
+        foreach ($tags as &$tag) {
+            if ($tag === '') {
+                $tag = '""';
+            } elseif (is_bool($tag)) {
+                $tag = ($tag ? "true" : "false");
+            } elseif (is_null($tag)) {
+                $tag = ("null");
+            }
+
```
Comment (gianarb) [inline src/InfluxDB/Point.php]: you can remove this line

Comment (eugenegp) [inline src/InfluxDB/Point.php]: @gianarb sure

File: src/InfluxDB/Point.php [hunk 2]
```diff
@@ -122,6 +122,15 @@ public function getTags()
      */
     public function setTags($tags)
     {
+        foreach ($tags as &$tag) {
+            if ($tag === '') {
```
Comment (sergeyklay) [inline src/InfluxDB/Point.php]: Hom about `$tag = '    ';`?

Comment (eugenegp) [inline src/InfluxDB/Point.php]: @sergeyklay this case already covered by `Point::addSlashes`, added test for this case

**Comment by eugenegp** [comment]
@gianarb  any problem with merging this PR?

**Comment by gianarb** [comment]
Just the red test, can you have a look? I can but next week

**Comment by eugenegp** [comment]
@gianarb fixed the test

**Comment by gianarb** [comment]
Thanks a lot

**Pull request merged**